pytorch_blade/src/torch-mlir/lib/Conversion/MhloPasses.cpp

```diff
@@ -55,6 +55,7 @@ void mlir::torch::createDiscTorchBackendToMhloBackendPipeline(
   pm.addNestedPass<func::FuncOp>(createDiscConvertTorchToMhloPass());
   pm.addNestedPass<func::FuncOp>(createConvertTorchToMhloPass(
       /*enableStaticShape*/ false, /*enableI32Index*/ true));
+  pm.addNestedPass<func::FuncOp>(createDiscLowerOperatorOpPass());
```

Comment:
what's the meaning of lower operator?

Reply:
It 'lower `OperatorOp`', not 'lower operator' 😂 ... Torch has an Op called `OperatorOp`, it is used to describe custom operators, see: https://pytorch.org/tutorials/advanced/torch_script_custom_ops.html

Reply:
Actually, `OperatorOp` is defined in https://github.com/llvm/torch-mlir/blob/main/include/torch-mlir/Dialect/Torch/IR/TorchOps.td#L832 used to represent those operators from PyTorch but have no dialect registered in Torch Dialect.